oauth/examples/sailfishoauthqmlexample/sailfishoauthqmlexample.qml

```diff
@@ -0,0 +1,116 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.OAuth 1.0
+
+ApplicationWindow {
+    id: window
+    initialPage: Component {
+        Page {
+            id: page
+
+            Rectangle {
+                id: topRect
+                width: parent.width
+                height: parent.height / 2
+                color: "lightsteelblue"
+
+                MouseArea {
+                    anchors.fill: parent
+                    property bool triggered: false
+                    onClicked: {
+                        if (!triggered) {
+                            triggered = true
+                            twitterOAuth.requestTemporaryToken()
+                        }
+                    }
+                }
+
+                OAuth10a {
+                    id: twitterOAuth
+
+                    property var xhr
+
+                    consumerKey: ""     // use your app's consumer_key value
+                    consumerSecret: ""  // use your app's consumer_secret value
+
+                    requestTokenEndpoint: "https://api.twitter.com/oauth/request_token"
+                    authorizeTokenEndpoint: "https://api.twitter.com/oauth/authorize"
+                    accessTokenEndpoint: "https://api.twitter.com/oauth/access_token"
+
+                    onErrorOccurred: console.log("Twitter OAuth1 Error: " + code + " = " + message)
+
+                    onReceivedTemporaryToken: {
+                        console.log("Got temporary token: " + oauthToken)
+                        authorizeInBrowser(oauthToken, oauthTokenSecret)
+                    }
+                    onReceivedAccessToken: {
+                        console.log("Got access token: " + oauthToken + "; " + oauthTokenSecret)
+                        xhr = authenticatedRequest("GET", "https://api.twitter.com/1.1/account/verify_credentials.json",
+                                                   oauth1.generateAuthorizationHeader(
+                                                           oauthToken, oauthTokenSecret,
+                                                           "GET", "https://api.twitter.com/1.1/account/verify_credentials.json"))
+                    }
+                }
+            }
+            Rectangle {
+                id: bottomRect
+                width: topRect.width
+                height: topRect.height
+                y: topRect.height
+                color: "lightgreen"
+
+                MouseArea {
+                    anchors.fill: parent
+                    property bool triggered: false
```

Comment:
```suggestion
                    property bool triggered
```
It is false by default

Reply:
Done, and same for the triggered property of the other mouse area